src/styles/style.css

```diff
@@ -0,0 +1,4 @@
+@import url(./reset.css);
+@import url(./blocks/card.css);
+@import url(./blocks/stars.css);
+@import url(./variables.css);
```

Comment:
Import variables.css before other CSS files to make sure variables are available when needed.